src/main/resources/licenses/apache-license-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
The license of all dependencies is **The Apache Software License, Version 2.0**

- commons-codec
- commons-logging
- fluent-hc
- httpclient
- httpcore
- jackson-annotationse
- Jackson-core
- jackson-databind